gitlab-ci.sample.yml

```diff
@@ -190,9 +192,9 @@ Distribute Packages:
   script:
     - pip install awscli
     - export AWS_DEFAULT_REGION="ap-southeast-2"
-    - if [[ $CI_CONFIG_TARBALL_TARGET != "AWS_BUCKET" ]]; then aws s3 cp build/*.tar.gz s3://${CI_CONFIG_TARBALL_TARGET}/${CI_PROJECT_NAME}/ ; fi
+    - if [[ $CI_CONFIG_TARBALL_TARGET != "AWS_BUCKET" ]]; then aws s3 cp build/*.tar.gz s3://${CI_CONFIG_TARBALL_TARGET}/${CI_CONFIG_BUILD_PROJECT}/ ; fi
     - if [[ $CI_CONFIG_TARBALL_TARGET != "AWS_BUCKET" ]]; then aws s3 rm s3://${CI_CONFIG_TARBALL_TARGET}/ --exclude "*" --include '${PROJECT_NAME}/${CI_PROJECT_NAME}-*.${CI_COMMIT_REF_SLUG}.*' --recursive ; fi
```

Comment:
@symric should this also be changed to use the new variable?

Reply:
Yes, copied wrong variable name

Reply:
Sweet, let me know when that's done and I can approve this for merge :+1:

Reply:
Can see that change coming through now. Looks good.